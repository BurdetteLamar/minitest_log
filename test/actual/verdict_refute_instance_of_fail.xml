<log>
  <summary_ verdicts='2' failures='2' errors='0'/>
  <section_ name='Test' method='verdict_refute_instance_of?' type='fail'>
    <section_ name='Args=[Integer, 0, &quot;Message 0&quot;]'>
      <verdict_ method='verdict_refute_instance_of?' outcome='failed' id='0' message='Message 0'>
        <expected_ class='Class' value='Integer'/>
        <actual_ class='Integer' value='0'/>
        <exception_ class='Minitest::Assertion' message='Expected 0 to not be an instance of Integer.'>
          <backtrace_/>
        </exception_>
      </verdict_>
    </section_>
    <section_ name='Args=[Symbol, :a, &quot;Message 1&quot;]'>
      <verdict_ method='verdict_refute_instance_of?' outcome='failed' id='1' message='Message 1'>
        <expected_ class='Class' value='Symbol'/>
        <actual_ class='Symbol' value=':a'/>
        <exception_ class='Minitest::Assertion' message='Expected :a to not be an instance of Symbol.'>
          <backtrace_/>
        </exception_>
      </verdict_>
    </section_>
  </section_>
</log>
