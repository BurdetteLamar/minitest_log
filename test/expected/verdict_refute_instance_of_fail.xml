<log>
  <summary verdicts='2' failures='2' errors='0'/>
  <section name='Test' method='verdict_refute_instance_of?' type='fail'>
    <section name='Args=[Integer, 0]'>
      <verdict method='verdict_refute_instance_of?' outcome='failed' id='0' message='Message 0'>
        <expected class='Class' value='Integer'/>
        <actual class='Integer' value='0'/>
        <exception class='Minitest::Assertion' message='Expected 0 to not be an instance of Integer.'>
          <backtrace/>
        </exception>
      </verdict>
    </section>
    <section name='Args=[Symbol, :a]'>
      <verdict method='verdict_refute_instance_of?' outcome='failed' id='1' message='Message 1'>
        <expected class='Class' value='Symbol'/>
        <actual class='Symbol' value=':a'/>
        <exception class='Minitest::Assertion' message='Expected :a to not be an instance of Symbol.'>
          <backtrace/>
        </exception>
      </verdict>
    </section>
  </section>
</log>
